libvast/src/system/start_command.cpp

```diff
@@ -85,6 +86,9 @@ caf::message start_command_impl(start_command_extra_steps extra_steps,
     return caf::make_message(std::move(bound_port.error()));
   auto listen_addr = std::string{host} + ':' + std::to_string(*bound_port);
   VAST_INFO_ANON("VAST node is listening on", listen_addr);
+  // Notify the service manager if it expects an update.
+  if (auto error = systemd::notify_ready())
+    return caf::make_message(std::move(error));
```

Comment:
Just to be extra safe, we should probably do this after starting the `signal_monitor` thread.